### ISSUE TITLE
Clarify Wording in HW1

### ISSUE THREAD

**Issue by WardLT** [WardLT/applied-ai-for-materials#15]
It wasn't clear whether you were getting `.coef_` from randomforest or LASSO.

**Comment by WardLT** [comment]
And be explicit about what size of test set to make

**Comment by WardLT** [comment]
Also change the size of the training set, while you're at it

**Comment by WardLT** [comment]
And mention that you can ignore `ConvergenceErrors`

**Comment by WardLT** [comment]
Also give an idea on the level of detail I expect
